src/edgepi/adc/edgepi_adc.py

```diff
@@ -383,8 +384,8 @@ def __get_calibration_values(self, adc_calibs: dict, adc_num: ADCNum) -> CalibPa
             raise ValueError("Cannot retrieve calibration values for channel in float mode")
 
         calib_key = mux_p.value if mux_n.code == CH.AINCOM else self.__get_diff_id(mux_p, mux_n)
-
-        calibs = adc_calibs.get(calib_key)
+        # TODO: fix this properly later
+        calibs = adc_calibs[calib_key]
```

Comment:
please explain in the TODO what exactly needs to be fixed : )

Reply:
ah, I forgot to remove that after adding extract_ch_dictionary method in eeprom_adc_module.py

Reply:
[f12061f](https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/357/commits/f12061fdd6ca21808c3d7e722733295a8d7de261)